ui/src/utils/api/configuration.ts

```diff
@@ -19,71 +19,72 @@
  * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
  *
  * OpenAPI spec version: 0.2.0
- * 
+ *
  *
  * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
  * https://openapi-generator.tech
  * Do not edit the class manually.
```

Comment:
Should it be under `.../generated`?

Reply:
Might be an old artifact that I missed when I introduced the `generated` directory for UI... is it still used?

Reply:
i just removed it. What about `custom.d.ts`? Is that still required too? That is in the same directory.

Reply:
TBH, I do not really know :)

Reply:
i''ll leave it for now and test/remove later :-) thanks